Exiled.Events/EventArgs/TurningOffLightsEventArgs.cs

```diff
@@ -0,0 +1,43 @@
+// -----------------------------------------------------------------------
+// <copyright file="TurningOffLightsEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs
+{
+    /// <summary>
+    /// Contains all information before turning off lights.
+    /// </summary>
+    public class TurningOffLightsEventArgs : System.EventArgs
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TurningOffLightsEventArgs"/> class.
+        /// </summary>
+        /// <param name="duration"><inheritdoc cref="Duration"/></param>
+        /// <param name="flickerableLightControllerHandler"><inheritdoc cref="FlickerableLightControllerHandler"/></param>
+        /// <param name="isAllowed"><inheritdoc cref="IsAllowed"/></param>
+        public TurningOffLightsEventArgs(FlickerableLightController flickerableLightControllerHandler, float duration, bool isAllowed = true)
+        {
+            FlickerableLightControllerHandler = flickerableLightControllerHandler;
+            Duration = duration;
+            IsAllowed = isAllowed;
+        }
+
+        /// <summary>
+        /// Gets the <see cref="FlickerableLightControllerHandler"/>.
```

Comment:
I think you forgot `Instance` after `Gets the see cref="FlickerableLightComtrollerHandler"/>`